eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -158,6 +158,12 @@ protected function configure()
                 InputOption::VALUE_OPTIONAL,
                 'Number of child processes to run in parallel for iterations, if set to "auto" it will set to number of CPU cores -1, set to "1" or "0" to disable',
                 'auto'
+            )->addOption(
+                'offset',
+                null,
+                InputOption::VALUE_OPTIONAL,
+                'Offset, useful when the script was interrupted and need to be resumed',
+                0
```

Comment:
Validation is missing here. Throw `InvalidArgumentException` when `offset < 0`.  For example:  https://github.com/ezsystems/ezplatform-kernel/blob/master/eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php#L193-L195 .